Flat/src/GeneralTree.cc

```diff
@@ -751,6 +762,8 @@ void GeneralTree::WriteTree(TTree *t) {
   for (auto p : ecfParams) { 
     TString ecfn(makeECFString(p));
     Book("fj1"+ecfn,&(fj1ECFNs[p]),"fj1"+ecfn+"/F");
+    if (monohiggs)
+      Book("fj1Gen"+ecfn,&(fj1GenECFNs[p]),"fj1Gen"+ecfn+"/F");
```

Comment:
This is where I should've made that comment about not always booking this branch.